tests/rules/test_key_ordering.py

```diff
@@ -147,3 +147,40 @@ def test_locale_accents(self):
                    'hais: true\n'
                    'haïr: true\n', conf,
                    problem=(3, 1))
+
+    def test_ignored_keys(self):
+        conf = 'key-ordering:\n  ignored-keys: ["n(am|omm)e", "^b.*$"]'
```

Comment:
Minor: With this regex specifically, it doesn't really serve any purpose to make it as complex as it is.

```suggestion
        conf = 'key-ordering:\n  ignored-keys: ["n(a|om)me", "^b"]'
```

It would also improve the clarity and utility of the tests if you broke the association with the beginning and ends of the words by including the matches in other parts of the string. ie. `first-name-here` instead of simply `name`.

Reply:
applied